### ISSUE TITLE
ID validation error in empty diagram

### ISSUE THREAD

**Issue by mschoe** [bpmn-io/bpmn-js-properties-panel#156]
when I change the ID in an empty diagram I get following error in the borowser consloe:
```
TypeError: Cannot read property 'attr' of null
    at InteractionEvents.js:144
    at invokeFunction (EventBus.js:457)
    at EventBus._invokeListener (EventBus.js:356)
    at EventBus._invokeListeners (EventBus.js:344)
    at EventBus.fire (EventBus.js:308)
    at ChangeSupport.js:34
    at invokeFunction (EventBus.js:457)
    at EventBus._invokeListener (EventBus.js:356)
    at EventBus._invokeListeners (EventBus.js:344)
    at EventBus.fire (EventBus.js:308)
```

#### Steps to reproduce

1) create a new diagram
2) delete the start event
3) try to modify the id of the process

The result is that the exception above is thrown and the modified id is not persisted.

**Comment by nikku** [comment]
Cannot reproduce. What is an "empty diagram"?

**Comment by romansmirnov** [comment]
An "empty diagram" is a diagram without any task or event inside. So, the steps to reproduce it are:
1) create a new diagram
2) delete the start event
3) try to modify the id of the process

The result is that the exception above is thrown and the modified id is not persisted.

**Comment by nikku** [comment]
This is actually a diagram-js bug. Gets fixed with bpmn-io/diagram-js#149.

**Comment by pedesen** [comment]
Fixed with https://github.com/bpmn-io/diagram-js/commit/90ca56f883dfa9b72feee782afbdcf5422899387
